examples/lora/sensor-lorawan/Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../../
+
+STACK_SIZE = 4096
+
+# Which files to compile.
+CXX_SRCS := $(wildcard *.cc)
+
+# Include the core RadioLib headers
+override CPPFLAGS += -isystem $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/src
+
+# Include the Tock specific headers
+override CPPFLAGS += -isystem $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/examples/NonArduino/Tock
+
+# Include the base of libtock-c to fix the libtock/ includes from RadioLib
+override CPPFLAGS += -I$(TOCK_USERLAND_BASE_DIR)/ -DRADIOLIB_CLOCK_DRIFT_MS=9
```

Comment:
Do we need `-I$(TOCK_USERLAND_BASE_DIR)`? I think that should already be happening.